providers/cncf/kubernetes/src/airflow/providers/cncf/kubernetes/decorators/kubernetes.py

```diff
@@ -62,7 +62,7 @@ class _KubernetesDecoratedOperator(DecoratedOperator, KubernetesPodOperator):
 
     # `cmds` and `arguments` are used internally by the operator
     template_fields: Sequence[str] = tuple(
-        {"op_args", "op_kwargs", *KubernetesPodOperator.template_fields} - {"cmds", "arguments"}
+        sorted({"op_args", "op_kwargs", *KubernetesPodOperator.template_fields} - {"cmds", "arguments"})
```

Comment:
Instead f sorting here and in the KubernetesJobOperator -- shouldn't we actually be sorting this on KubernetesPodOperator -- otherwise I think using KPO directly would still result in hash changes.